rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/dataspace/dto/ListFile.java

```diff
@@ -36,15 +36,16 @@
  */
 package org.ow2.proactive_grid_cloud_portal.dataspace.dto;
 
-import java.util.List;
-
 import org.codehaus.jackson.map.annotate.JsonSerialize;
 
+import java.util.List;
+
 
 @JsonSerialize(include = JsonSerialize.Inclusion.NON_EMPTY)
 public class ListFile {
     private List<String> directories;
     private List<String> files;
+    private List<String> all;
```

Comment:
I think that you can remove the attribute "all" and the setter "setAll" and change getAll by getDirectoriesAndFiles where you add both list into a new one with addAll method from List class. I think that setAll and all attribute are usefull only when you can't distinguish files and directories. However in the commit, setAll is used at the same time that setDirectories and setFile.